### PR TITLE
Split libvirt tftp configuration into another task

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: playbooks/provision_test_box.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+- hosts: all
+  become: true
+  roles:
+    - role: libvirt_install
+    - role: vagrant_install
```
Comment (ehelms) [inline playbooks/provision_test_box.yml]: I think the vagrant install should be it own PR.

File: roles/libvirt_configure/tasks/main.yml [hunk 2]
```diff
@@ -63,9 +32,6 @@
 - name: 'copy syslinux files to /var/lib/tftpboot'
   shell: cp /usr/share/syslinux/{pxelinux.0,menu.c32,chain.c32} /var/lib/tftpboot
 
-- name: 'chown /var/lib/tftpboot for foreman-proxy user'
```
Comment (ehelms) [inline roles/libvirt_configure/tasks/main.yml]: Why remove this?

**Comment by ehelms** [comment]
The naming of the two roles does not leave me with a sense of obvious as to what they do. I would think of it more like the following:

Option 1:

 * libvirt role
 * variable for libvirt_tftp that configures the tftp part optionally

Option 2:

 * libvirt role
 * libvirt_tftp role

File: roles/libvirt_configure/tasks/main.yml (hunk 2 above)
Comment (zjhuntin) [reply roles/libvirt_configure/tasks/main.yml]: Wasn't meaning to, I removed it temporarily to get it working on another machine. I'll replace it.

File: playbooks/katello_provisioning.yml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 - hosts: all
   become: true
+  vars:
+    tftp: true
```
Comment (ehelms) [inline playbooks/katello_provisioning.yml]: Best practice is to scope variables based on role. For example, this would become `libvirt_tftp`. Also, we should set a default in defaults/main.yml

**Comment by zjhuntin** [comment]
@ehelms Updated!

**Comment by ehelms** [comment]
@zjhuntin Needs a rebase please

**Comment by zjhuntin** [comment]
@ehelms Rebased
